languages/ruby/lib/oso/polar/ffi/message.rb

```diff
@@ -1,40 +0,0 @@
-# frozen_string_literal: true
```

Comment:
@gj I can back out these changes. But I couldn't see what these were achieving other than obscuring what the types were? These are all just strings at the end of the day. Any objections/rationale I'm missing?

Reply:
works for me